### ISSUE TITLE
403: Too many unsuccessful login attempts. Please restart transmission-daemon.

### ISSUE THREAD

**Issue by amandamana** [transmission/transmission#1762]
The remote access web interface when set to require authentication stops working after a few days, apparently due to erroneously detecting a brute force attack. I've seen this using a normal install of version 3.00 on macOS in a secure LAN, where actual unsuccessful login attempts are extremely unlikely.

This issue was seen as far back as [Oct 2020](https://forum.transmissionbt.com/viewtopic.php?t=20576).

More information in this [longer thread](https://github.com/haugene/docker-transmission-openvpn/issues/1493) of both docker and normal installations.

**Comment by JBPhoenix** [comment]
Can confirm - quite annoying at times.
There seem to be changes in master (see pull https://github.com/transmission/transmission/pull/371) that allow configuration and the ability to disable the brute force protection but no official release for that yet.

**Comment by dpkonofa** [comment]
Can also confirm. Logins are limited to IPs on my machine so I know it's not external unsuccessful logins.

**Comment by ambipur76** [comment]
I have upgraded my debian from buster to bullseye and this induced transmission to be upgraded from 2.94 to 3.0 . Previously with transmission 2.94 or even older versions I did not have this issue. Could anybody support us somehow, to avoid this issue?

**Comment by davidXire** [comment]
Any news on that ?

**Comment by ambipur76** [comment]
Sorry, I have forgotten to reply on this topic. In my case I had a perl script triggered by cron. This script tried to gain up- and download statistics information into an rrdtool database. Something was wrong with that script, I did not bother with it at least 1 year. I simply deactivated that script because it was not really important for me and since some weeks transmission 3.0 is working fine for me.
